src/mon/MonCommands.h

```diff
@@ -175,6 +175,12 @@ COMMAND("auth get-or-create " \
 	"name=caps,type=CephString,n=N,req=false", \
 	"add auth info for <entity> from input file, or random key if no input given, and/or any caps specified in the command", \
 	"auth", "rwx", "cli,rest")
+COMMAND("auth fs " \
```

Comment:
My preference would be `fs authorize` -- feels like it's more naturally going to be used alongside all the other `fs` commands, even if it's implemented in authmonitor.